src/DuetNG/DueXn.cpp

```diff
@@ -199,7 +222,19 @@ namespace DuetExpansion
 	// Return the name of the additional expansion board, or nullptr if no expansion board
 	const char* _ecv_array null GetAdditionalExpansionBoardName()
 	{
-		return (additionalIoExpanderPresent) ? "SX1509B expander" : nullptr;
+		if (additionalIoExpanderPresent && additionalIoExpanderPresent2)
+		{
+			return "SX1509B expander (0x71), SX1509B expander (0x3E)";
```

Comment:
This could be different, depending on how you would like it to display the presence of two SX1509Bs.

Perhaps: 

`return "SX1509B expander (0x71 & 0x3E)";`

Would look better.